db/alias_manager_test.go

```diff
@@ -25,17 +25,17 @@ func TestAliasManagement(t *testing.T) {
 		assert.Nil(t, err)
 	})
 
-	// Request unique ID numbers from the counter
-	t.Run("GetID", func(t *testing.T) {
-		for i := 0; i < 32; i++ {
-			u, err := d.GetID(context.Background(), "classes_alias")
-			assert.Nil(t, err)
+	// // Request unique ID numbers from the counter
+	// t.Run("GetID", func(t *testing.T) {
+	// 	for i := 0; i < 32; i++ {
+	// 		u, err := d.GetID(context.Background(), "classes_alias")
+	// 		assert.Nil(t, err)
 
-			t.Logf("Unique ID + hashing:\t\t%d\n", u)
-			w := tinycrypt.GenerateWord24(uint64(u))
-			wid := strings.Join(w, ",")
-			t.Logf("Unique ID + hashing + words:\t%s\n======", wid)
-		}
-	})
+	// 		t.Logf("Unique ID + hashing:\t\t%d\n", u)
+	// 		w := tinycrypt.GenerateWord24(uint64(u))
+	// 		wid := strings.Join(w, ",")
+	// 		t.Logf("Unique ID + hashing + words:\t%s\n======", wid)
+	// 	}
+	// })
```

Comment:
Don't be afraid to remove this (and some of the other commented blocks) altogether.